console_windows.go

```diff
@@ -62,8 +64,10 @@ func (m *master) initStdios() {
 			windows.SetConsoleMode(m.err, m.errMode)
 		}
 	} else {
-		fmt.Printf("failed to get console mode for stderr: %v\n", err)
+		errs = append(errs, fmt.Errorf("unable to get console mode for stderr: %w", err))
 	}
+
+	return errs
```

Comment:
Why even collect the errors if we're just going to ignore them in the only spot we call this?

Reply:
Oh I didn't notice that. Actually since newMaster() can return an error, does it make sense to return this error from the function?

Reply:
Possibly. This also just seems like something we could be able to solve with opening CONOUT/IN to get the console handle if it's been redirected: 
1. https://github.com/PowerShell/PowerShell/blob/2ec7aea2ec18d8729dfd9d57c321b05dc97a6737/src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs#L587
2. https://learn.microsoft.com/en-us/windows/console/console-handles

To unblock the issues this was causing maybe we just ignore failures here and we can work on supporting redirected handles in a follow up

Reply:
I'm also kinda curious what you get from this, you can't really do anything with the object (because they all almost all call SetConsoleMode), so you just have Read/Write to use

Reply:
I decided to collect them because it felt wrong to just silently discard these errors at that stage. It should be up to the caller to decide how to deal with them - even though our only caller is `newMaster`. As I've pointed out in the comment of that method, it's not unusual to encounter errors here (when in/out is redirected), so discarding still feels appropriate here.

Reply:
> It should be up to the caller to decide how to deal with them

I agree if we had > 1 caller, it doesn't seem like that'll be the case though. Lets discard for now and maybe we can revisit this if we need to init anywhere else. Also sorry for taking so long to get to these, and thanks for the contribution!

Reply:
Done! PTAL!

Reply:
Thanks!